### PR TITLE
Nerfs Sonic Speed Wheelchair

### PR DESCRIPTION
**Pull request by TheFurryFeline**
Porting a few things from CHOMPstation, essentially to make it so that you're not going at Mach 3.

### CODE REVIEW

**Comment by MisterLayne** [comment]
This also stops you from sumo-slamming the person against doors and windows.

**Comment by Anewbe** [comment]
I don't see any code that'd do that, @MisterLayne ?

Also, out of curiosity, @TheFurryFeline , why specifically are you doing this as implemented?

**Comment by TheFurryFeline** [comment]
Ack, sorry for being silent up to now, @Anewbe. Basically, it's pretty much what's on the tin, makes it so that you're not going like a.... Well, maniac on wheels. Ideally,  giving a few thoughts to it now, this would prevent characters from outrunning virtually anyone that are running.

**Comment by TheFurryFeline** [comment]
Additionally, I'm wanting this to also be implemented on VOREstation but with the filename required, I'm over up here so it'll come out with a mirror if/when this is merged.

**Comment by Anewbe** [comment]
Sorry for the vagueness on my part; my more specific question was why in that specific part of the code, rather than something like movement?

**Comment by TheFurryFeline** [comment]
Ah, if I placed it in an odd area, then sorry. I pretty much just copy-pasted from another codebase into this, keeping the lines in their same places. I'm still rather new to coding.

**Comment by Anewbe** [comment]
No worries.
Do you know how to test your code, at least?

**Comment by TheFurryFeline** [comment]
I've already seen this code in action on CHOMPstation, actually, and with it, it has a noticeable difference in movement speed for wheelchairs and on VOREstation, it's still supersonic speed without this being part of the file.

**Comment by Anewbe** [comment]
I don't disbelieve you, but I'd rather you actually test this code, as it applies specifically to Polaris.
I've got some concerns about the pushing wheelchairs as this is implemented, and I'd much rather hear from you if that does something weird, than have to find out for myself in a few months when I've forgotten about this PR, if that makes sense?

**Comment by TheFurryFeline** [comment]
Fair enough, I'll clone the codebase and then open up a test client, try out the chair. Then change the file and compare the speed rate.

**Comment by Anewbe** [comment]
Thank you

**Comment by TheFurryFeline** [comment]
Huh..... Took a while for me to get around to it but without this in the code, it seems that wheelchairs aren't super fast here. Unless the tickrate's been changed here, as well?

**Comment by Anewbe** [comment]
Tick rate is variable, ours hasn't been touched in a while.

**Comment by TheFurryFeline** [comment]
Alright, got the .dme open, putting the lines in to test to see if there's any issues.

**Comment by TheFurryFeline** [comment]
And again sorry for the delays, but it seems to work nicely. No issues that I noticed. :> As for pushing the wheelchairs, seems to be fine as well. Spawned human in, totally didn't bump 'em into walls and windows, and speed was normal as well as I could tell.

File: code/game/objects/structures/stool_bed_chair_nest/wheelchair.dm [hunk 1]
```diff
@@ -29,6 +30,13 @@
 
 /obj/structure/bed/chair/wheelchair/relaymove(mob/user, direction)
 	// Redundant check?
+ 	
+	var/calculated_move_delay
+	calculated_move_delay += 2 //TheFurryFeline: nerfs speed so you don't go like Sonic. >W>
```
Comment (SpadesNeil) [inline code/game/objects/structures/stool_bed_chair_nest/wheelchair.dm]: **>W>**

**Comment by SpadesNeil** [comment]
Seriously though this is a good tweak.

**Pull request merged**